style.css

```diff
@@ -9,15 +11,49 @@
   }
 
   .main {
+    border-top: none;
+    width: auto;
+    height: auto;
+  }
+
+  .header-section {
     background-image: url(img/header-shapes.png);
     background-size: cover;
     border-radius: 0 0 10px 70px;
     border: 1px solid #dfe1e6;
     border-top: none;
+    width: 365px;
+    height: 626px;
+  }
+
+  .myself-section {
+    background-color: #fff;
+    border-radius: 10px 70px 0 0;
+    border: 1px solid #dfe1e6;
+    border-top: none;
+    margin-top: 10%;
     width: auto;
     height: 626px;
   }
```

Comment:
- [ ] [ OPTIONAL] Great work styling your About myself section 💯. However, I noticed it is overlapping with the section from above🤓. Kindly consider applying a `margin-top: 200px;` CSS property to this section in order to have the correct space and stick as much as possible to the Figma design. 🔧